src/Symfony/Component/VarDumper/Caster/IntlCaster.php

```diff
@@ -123,6 +123,58 @@ public static function castIntlTimeZone(\IntlTimeZone $c, array $a, Stub $stub,
         return self::castError($c, $a);
     }
 
+    public static function castIntlCalendar(\IntlCalendar $c, array $a, Stub $stub, $isNested, $filter = 0)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'type' => $c->getType(),
+            Caster::PREFIX_VIRTUAL.'first_day_of_week' => $c->getFirstDayOfWeek(),
+            Caster::PREFIX_VIRTUAL.'minimal_days_in_first_week' => $c->getMinimalDaysInFirstWeek(),
+            Caster::PREFIX_VIRTUAL.'repeated_wall_time_option' => $c->getRepeatedWallTimeOption(),
+            Caster::PREFIX_VIRTUAL.'skipped_wall_time_option' => $c->getSkippedWallTimeOption(),
+            Caster::PREFIX_VIRTUAL.'time' => $c->getTime(),
+            Caster::PREFIX_VIRTUAL.'type' => $c->getType(),
+            Caster::PREFIX_VIRTUAL.'in_daylight_time' => $c->inDaylightTime(),
+            Caster::PREFIX_VIRTUAL.'is_lenient' => $c->isLenient(),
+        );
+
+        if ($filter & Caster::EXCLUDE_VERBOSE) {
+            ++$stub->cut;
+
+            return self::castError($c, $a);
+        }
+
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'time_zone' => $c->getTimeZone(),
```

Comment:
you're going to be an expert in writing caster :)
here, I'd suggest to remove the previous "if" and replace it with this is the big array above:
`Caster::PREFIX_VIRTUAL.'time_zone' => ($filter & Caster::EXCLUDE_VERBOSE) ? new CutStub($c->getTimeZone()) : $c->getTimeZone()`
(same below)